test/modules/fundingManager/bondingCurve/utils/mocks/FM_BC_Bancor_Redeeming_VirtualSupplyV1Mock.sol

```diff
@@ -104,6 +107,22 @@ contract FM_BC_Bancor_Redeeming_VirtualSupplyV1Mock is
             _calculateNetAndSplitFees(_totalAmount, _protocolFee, _workflowFee);
     }
 
+    function call_handleIssuanceTokenAfterBuy(address _receiver, uint _amount)
+        external
+    {
+        _handleIssuanceTokenAfterBuy(_receiver, _amount);
+    }
+
+    function call_handleCollateralTokenAfterSell(
+        address _receiver,
+        uint _amount
+    ) external {
+        _handleCollateralTokenAfterSell(_receiver, _amount);
+    }
+
+    //--------------------------------------------------------------------------
```

Comment:
Missing space between `//` and `---`
(maybe just update the whole file if you are at it?)